HomeAutomationServer/Program.cs

```diff
@@ -83,11 +83,11 @@ private static void Main(string[] args)
                 return;
         }
 
-        RunServicesAsync().GetAwaiter().GetResult();
+        await RunServicesAsync().ConfigureAwait(false);
 
         while (true)
         {
-            Thread.Sleep(5000);
+            await Task.Delay(5000).ConfigureAwait(false);
```

Comment:
```suggestion
            await Task.Delay(5000).ConfigureAwait(true);
```
As the while loop runs forever, it should not do so on a thread pool thread